apps/build_disk_index.cpp

```diff
@@ -9,6 +9,7 @@
 #include "math_utils.h"
 #include "index.h"
 #include "partition.h"
+#include "../include/program_options_utils.hpp"
```

Comment:
should just have to do `#include "program_option_utils.hpp" - the include directories are already pulled in as the root dir for includes

Reply:
Thank you.  Removed the superfluous paths and spot-checked a few of the compiled tools to make sure they still launch.